enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/subscription/SubscriptionDataStoreImpl.java

```diff
@@ -546,7 +546,8 @@ public List<SubscriptionPolicy> getMatchingSubscriptionPolicies(String policyNam
         if (StringUtils.isEmpty(policyName)) {
             subscriptionPolicies.addAll(this.subscriptionPolicyMap.values());
         } else {
-            SubscriptionPolicy policy = this.getSubscriptionPolicyByName(policyName);
+            SubscriptionPolicy policy = this.getSubscriptionPolicyByOrgIdAndName(APIConstants.SUPER_TENANT_DOMAIN_NAME,
```

Comment:
Why are we hardcoding SUPER_TENANT_DOMAIN_NAME here?

Reply:
In other words:
- Where is this method being used? `getMatchingSubscriptionPolicies `
- Is it correct to assume the org is always SUPER_TENANT_DOMAIN_NAME when calling this method? i.e Why don't we need to pass orgId from a method variable?

Reply:
This method is relevant to the retrieval of subscription policies that are keeping in the Enforcer. As per the team discussions this was added during the JMS based event retrieval.

Reply:
@slahirucd7, this looks like a simple fix, Shall we do this?

We can add a new query param key, `organization` and extract it as here we've done for the policy name.

https://github.com/wso2/product-microgateway/blob/8371e29172a93a44dcd6fad28dcdab7320451d23/enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/admin/handlers/ThrottlingPolicyRequestHandler.java#L38-L47

/throttling_policies/application?name=Gold&orgnization=123456789

Then let's add the org param to this function